### ISSUE TITLE
CF Deployment Manifest Should Populate CAPI /v2/info Fields

### ISSUE THREAD

**Issue by davewalter** [cloudfoundry/cf-deployment#813]
### What is this issue about?

We received a [slack interrupt](https://cloudfoundry.slack.com/archives/C2U7KA7M4/p1566235839033800) commenting on the fact that the CAPI `build` property was not being set by default. I am opening this issue to foster discussion about what we could/should be configuring by default and how it would be kept up to date.

### What version of [cf-deployment](https://github.com/cloudfoundry/cf-deployment/releases) are you using?

v11.0.0

### Please include the `bosh deploy...` command, including all the operations files (plus any experimental operation files you're using):

```
bosh deploy cf-deployment.yml
```

### Please provide output that helps describe the issue:

```
$ cf curl /v2/info
{
   "name": "",
   "build": "",
   "support": "",
   "version": 0,
   "description": "",
   "min_cli_version": null,
   "min_recommended_cli_version": null,
   "api_version": "2.140.0",
   "osbapi_version": "2.15",
...
}
```

### What IaaS is this issue occurring on?

N/A

### Is there anything else unique or special about your setup?

N/A

**Comment by davewalter** [comment]
What would you want the `build` property to be set to? I can see setting the `name` property to `cf-deployment` and the `version` property to `v11.0.0`, but build is a little harder. I was originally thinking about the `commit sha`, but that would never actually be 100% accurate due to the commit needed to change the value. It could match the `version`, but that seems a little redundant. Opinions welcome.

**Comment by davewalter** [comment]
Actually, we will need to use `build` for the version number as [version is an integer](https://github.com/cloudfoundry/cloud_controller_ng/blob/2523891f348526b5906b06b1b665696c47b4f581/lib/cloud_controller/config_schemas/api_schema.rb#L19) but cf-deployment's versions are semver.

**Comment by andrew-edgar** [comment]
right it was mainly the "build" that we used to have to override but now making it the same as the cf-deployment version seems good to us.  as for "version" yeah good question I think making it just the major release number might work?

**Comment by ctlong** [comment]
🎉 This feature was released in the latest version of CF-D: v32.11.0. 🚀
